yb-voyager/src/srcdb/ora2pg_export_data.go

```diff
@@ -63,6 +63,14 @@ func ora2pgExportDataOffline(ctx context.Context, source *Source, exportDir stri
 	defer utils.WaitGroup.Done()
 
 	configFilePath := filepath.Join(exportDir, "temp", ".ora2pg.conf")
+	customConfigFilePath := filepath.Join(exportDir, "metainfo", "base_ora2pg.conf")
+	if utils.FileOrFolderExists(customConfigFilePath) {
+		utils.PrintAndLog("Using custom base_ora2pg.conf for data migration.")
+		source.PopulateOra2pgConfigFile(customConfigFilePath)
+		configFilePath = customConfigFilePath
+	} else {
+		source.PopulateOra2pgConfigFile(configFilePath)
+	}
```

Comment:
This is too complicated. All we need to do is create, if it doesn't exists, the base_ora2pg.conf (from the value of the `Ora2pgConfigFile` variable). The `PopulateOra2pgConfigFile()` always takes the full path of the `temp/.ora2pg.conf`; loads the contents from the `base_ora2pg.conf`; and generates the `temp/.ora2pg.conf` after adding all the migration parameters to the contents loaded from the base_ora2pg.conf.